tests/unit/core/model/entity/test_segmentation.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Unit tests for segmentation model entity."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import pytest
+import torch
+from omegaconf import OmegaConf
+from otx.core.data.entity.base import ImageInfo
+from otx.core.data.entity.segmentation import SegBatchDataEntity
+from otx.core.model.entity.segmentation import MMSegCompatibleModel
+
+if TYPE_CHECKING:
+    from omegaconf.dictconfig import DictConfig
+
+
+class TestOTXSegmentationModel:
+    @pytest.fixture()
+    def config(self) -> DictConfig:
+        return OmegaConf.load("src/otx/recipe/segmentation/segnext_s.yaml")
+
+    @pytest.fixture()
+    def model(self, config) -> MMSegCompatibleModel:
+        return MMSegCompatibleModel(config.model.otx_model.config)
+
+    @pytest.fixture(autouse=True)
+    def data_entity(self) -> SegBatchDataEntity:
+        img_size = (240,240)
+        return SegBatchDataEntity(1, [torch.rand(img_size)],
+                                        [ImageInfo(0, img_size, img_size, img_size, img_size)],
+                                        [torch.rand(img_size)])
```

Comment:
Here is the duplication I mentioned in https://github.com/openvinotoolkit/training_extensions/pull/2715/files#r1423387124

Reply:
Done [77733bb](https://github.com/openvinotoolkit/training_extensions/pull/2715/commits/77733bb3d29399f1d20d273bb3b1011e7c2defff)